hazelcast/src/main/java/com/hazelcast/internal/monitor/impl/PerIndexStats.java

```diff
@@ -245,6 +255,10 @@ public IndexOperationStats createOperationStats() {
      */
     long getMemoryCost();
 
+    void updateMemoryCost(long delta);
```

Comment:
Missing javadoc

Reply:
Addressed in https://github.com/hazelcast/hazelcast/pull/25208/commits/77834391a38304ebe65aecff3ca4f1115994d024